policy/modules/system/systemd.te

```diff
@@ -237,6 +237,10 @@ files_type(systemd_zram_generator_conf_t)
 allow systemd_zram_generator_t systemd_zram_generator_conf_t:file read_file_perms;
 # domain for a generic generator
 systemd_generator_template(systemd_generic_generator)
+# User domains
+allow userdomain systemd_unit_file_t:dir manage_dir_perms;
+allow userdomain systemd_unit_file_t:file manage_file_perms;
+allow userdomain systemd_unit_file_t:lnk_file manage_lnk_file_perms;
```

Comment:
Types/attributes from other modules need to be accessed via interfaces. Anyway allowing any user type manage systemd_unit_file_t does not look good.